apps/condo/domains/billing/schema/BillingCategory.js

```diff
@@ -23,8 +23,10 @@ const BillingCategory = new GQLListSchema('BillingCategory', {
     plugins: [uuided(), versioned(), tracked(), softDeleted(), dvAndSender(), historical()],
     access: {
         read: access.canReadBillingCategories,
-        create: access.canManageBillingCategories,
-        update: access.canManageBillingCategories,
+        // NOTE: These categories are explicitly created in 0121 migration.
```

Comment:
it looks like we don't need to store it in database

Reply:
We decided to return an ability to create or update billing categories